inst/templates/package/testing_results_package.Rmd

```diff
@@ -0,0 +1,36 @@
+---
+title: "Testing Results"
+output:
+  word_document: default
+  pdf_document:
+    latex_engine: xelatex
+number_sections: yes
+params:
+  product_name: "package"
+  version: "v1.0.0"
+  auto_tests: ""
+  man_tests: ""
+  val_info: ""
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(results = "asis", echo=FALSE)
+library(knitr)
+library(flextable)
+```
+
+# Testing: `r params$product_name` `r params$version`
+
+## Scope
+The purpose of this Validation Testing document is to define the conditions for test execution and present the test results. All tests are specified and linked to release candidate user stories as numbered issues in the Requirements Specification document.
+
+## Automated Tests Results
+```{r}
+format_auto_tests_results(params$auto_tests, params$val_info)
+```
+
+
+```{r}
```

Comment:
Same as above, remove this manual tests section. If we need to put this stuff back in at some point later we can, but I would be pretty surprised if any packages end up having truly "manual" tests that require screenshots, etc.